onnx/defs/traditionalml/utils.h

```diff
@@ -0,0 +1,27 @@
+#include "onnx/defs/schema.h"
+#include "onnx/defs/shape_inference.h"
+
+namespace ONNX_NAMESPACE {
+
+void AssertAttributeProtoType(
```

Comment:
Can we make this name more explicit? It checks the size of an array.

Reply:
Sure, have done so.